src/Magnum/Platform/Sdl2Application.cpp

```diff
@@ -378,12 +378,11 @@ void Sdl2Application::mainLoop() {
                 event.type == SDL_MOUSEBUTTONDOWN ? mousePressEvent(e) : mouseReleaseEvent(e);
             } break;
 
-            case SDL_MOUSEWHEEL:
-                if(event.wheel.y != 0) {
-                    MouseEvent e(event.wheel.y > 0 ? MouseEvent::Button::WheelUp : MouseEvent::Button::WheelDown, {event.wheel.x, event.wheel.y}, 0);
-                    mousePressEvent(e);
-                } break;
-
+            case SDL_MOUSEWHEEL: {
+                MouseEvent e(event.wheel.y > 0 ? MouseEvent::Button::WheelUp : MouseEvent::Button::WheelDown, {event.wheel.x, event.wheel.y}, 0);
+                mousePressEvent(e);
```

Comment:
Aargh. Yes. Correct.

... probably the sanest thing to do is to introduce the scroll event and drop this. :D